src/main/java/com/uipath/uipathpackage/util/ActualVersion.java

```diff
@@ -0,0 +1,23 @@
+package com.uipath.uipathpackage.util;
+
+public class ActualVersion {
+    private final int major;
+    private final int minor;
+
+    public ActualVersion(int major, int minor) {
+        this.major = major;
+        this.minor = minor;
+    }
+
+    public int getMajor() {
+        return major;
+    }
+
+    public int getMinor() {
+        return minor;
+    }
+
+    public boolean supportsNewTelemetry() {
```

Comment:
Does this mean, we will only support telemetry from 23.10?

Reply:
Yes.
Older CLIs crash if are called with params like PipelineCorrelationId